HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 History
 =======
 
+Current
```

Comment:
Hey @vladcalin 

> I think the version should be updated by you when you decide it is the moment for a new release, mainly because you are the core developer of this library.

It's not like I am some BDFL man 😄 

I am open to discussion about things. So please feel free to agree/disagree with me on things 👍 

Merging your PR then. Will update the version after that. Sounds good?